warehouse/models/mart/gtfs/fct_hourly_rt_feed_files.sql

```diff
@@ -64,17 +79,19 @@ fct_hourly_rt_feed_files AS (
         url_index.base64_url,
         url_index.type AS feed_type,
 
-        pivot_hourly_totals.* EXCEPT (dt, base64_url, feed_type),
+        daily_totals.file_count_day,
+        daily_totals.gtfs_dataset_key,
 
-        url_map.gtfs_dataset_key
+        pivot_hourly_totals.* EXCEPT (dt, base64_url, feed_type),
 
     FROM int_gtfs_rt__daily_url_index AS url_index
     LEFT JOIN pivot_hourly_totals
         ON url_index.dt = pivot_hourly_totals.dt
             AND url_index.base64_url = pivot_hourly_totals.base64_url
             AND url_index.type = pivot_hourly_totals.feed_type
```

Comment:
@lauriemerrell ah so is this join on `feed_type` unnecessary then too?

Reply:
Yes I think so